opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/FeatureInstallKarafIT.java

```diff
@@ -0,0 +1,150 @@
+        installFeature("json-lib");
+        installFeature("lmax-disruptor");
+        installFeature("net.java.dev.jna");
+        // Causes the test to die?
+        //installFeature("onms-camel-elasticsearch");
+        //installFeature("opennms-activemq-config");
+        //installFeature("opennms-activemq");
+        //installFeature("opennms-activemq-dispatcher-config");
+        //installFeature("opennms-activemq-dispatcher");
+        // Causes the test to die?
+        //installFeature("opennms-amqp-alarm-northbounder");
+        // OSGi dependency problems: javax.transaction
+        //installFeature("opennms-amqp-event-forwarder");
+        // OSGi dependency problems: javax.transaction
+        //installFeature("opennms-amqp-event-receiver");
+        //installFeature("opennms-collection-api");
+        // OSGi dependency problems: javax.transaction
+        //installFeature("opennms-collection-persistence-rrd");
+        installFeature("opennms-config-api");
+        installFeature("opennms-config");
+        installFeature("opennms-config-jaxb");
+        installFeature("opennms-core-camel");
+        installFeature("opennms-core-daemon");
+        installFeature("opennms-core-db");
+        installFeature("opennms-core");
+        installFeature("opennms-core-web");
+        installFeature("opennms-dao-api");
+        installFeature("opennms-dao");
+        // Minion-only feature
+        //installFeature("opennms-discoverer");
+        //installFeature("opennms-discovery-daemon");
+        installFeature("opennms-discovery");
+        // Minion-only feature
+        //installFeature("opennms-discovery-distPollerDaoMinion");
+        //installFeature("opennms-elasticsearch-event-forwarder");
+        installFeature("opennms-events-api");
+        installFeature("opennms-events-daemon");
+        installFeature("opennms-events-traps");
+        installFeature("opennms-icmp-api");
+        installFeature("opennms-icmp-jna");
+        installFeature("opennms-icmp-jni");
+        installFeature("opennms-icmp-jni6");
+        installFeature("opennms-javamail");
+        installFeature("opennms-model");
+        installFeature("opennms-poller-api");
+        // OSGi dependency problems
+        //installFeature("opennms-provisioning");
+        // OSGi dependency problems
+        //installFeature("opennms-reporting");
+        installFeature("opennms-rrd-api");
+        installFeature("opennms-rrd-jrobin");
+        installFeature("opennms-snmp");
+        installFeature("opennms-syslogd");
+        //installFeature("opennms-syslogd-handler-default");
+        installFeature("opennms-syslogd-handler-kafka");
+        installFeature("opennms-syslogd-handler-minion");
+        // Syslog listeners can only be installed one at a time
+        //installFeature("opennms-syslogd-listener-camel-netty");
+        //installFeature("opennms-syslogd-listener-javanet");
+        //installFeature("opennms-syslogd-listener-nio");
```

Comment:
I just tried to add these and there's an issue: currently the Syslogd features rely on minion-core-api  because of the reused ActiveMQ component. We need to write a new test inside the project that constructs the Minion features repo for Syslogd that performs these tests. I'd like to defer that until after this PR is merged.